Add a quiet parameter to get_census()

atheriel opened the pull request:
This is available for all other functions, so it's nice to have it on the main one. This should also reduce some of the noisy output in vignettes.

I've also fixed some unsuppressed output in `list_census_vectors()`.

dshkol commented:
These vignettes will keep failing without putting in an API key. How to do that for vignettes to run in Travis without exposing it publicly?

mountainMath commented:
Should not be an issue in Travis, I added an API key as environment variable. Some of the vignettes fail right now because we changed the default geo_format. That is easy to fix. Others might fail because of the caching issues, not sure.

atheriel commented:
They fail for a variety of minor reasons. But I can confirm from my testing
in the other PR that the API key is working.

On Wed, Sep 6, 2017 at 3:14 AM Jens <notifications@github.com> wrote:

> Should not be an issue in Travis, I added an API key as environment
> variable. Some of the vignettes fail right now because we changed the
> default geo_format. That is easy to fix. Others might fail because of the
> caching issues, not sure.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/mountainMath/cancensus/pull/81#issuecomment-327396456>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABYZhmBXtRyd9iHpZm3OQrqOAU--xWmYks5sfkZhgaJpZM4PNzhr>
> .
>
